src/microbe_stage/editor/tooltips/SelectionMenuToolTip.cs

```diff
@@ -296,6 +314,16 @@ public void WriteMembraneModifierList(MembraneType referenceMembrane, MembraneTy
         }
     }
 
+    [RunOnKeyDown("help", Priority = int.MaxValue)]
+    public bool OpenMoreInfo()
+    {
+        if (!Visible || thriveopediaPageName == null)
+            return false;
+
+        ThriveopediaManager.OpenPage(Name);
```

Comment:
Shouldn't this use `thriveopediaPageName` here? I think I'll correct this while fixing the merge conflict to merge this today.